frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -244,6 +248,7 @@ export const ImageManifestVulnTableRow: RowFunction<ImageManifestVuln> = ({
         {Object.keys(obj.status.affectedPods).length}
       </TableData>
       <TableData className={tableColumnClasses[4]}>{obj.status.fixableCount || 0}</TableData>
+      <TableData className={tableColumnClasses[5]}>{totalCount(obj)}</TableData>
       <TableData className={tableColumnClasses[4]}>
```

Comment:
```suggestion
      <TableData className={tableColumnClasses[6]}>
```

and can you fill the empty tableColumnClasses with some `tableColumnClasses` content. We have a lot of `const tableColumnClasses` in our code which works better than the current one?! Also if this is not perfect this would be a good step into the right direction, or wdyt?

Reply:
Added tableColumnClasses.